src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -22,7 +22,7 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
     <Using Include="Android.OS.Bundle" Alias="Bundle" Platform="Android" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(EnableDefaultAndroidItems)' == 'true' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) ">
+  <ItemGroup Condition=" '$(MonoAndroidResourcePrefix)' != '' and '$(EnableDefaultAndroidItems)' == 'true' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) ">
     <!-- Default Resource file inclusion -->
     <!-- https://developer.android.com/guide/topics/resources/providing-resources -->
     <AndroidResource Include="$(MonoAndroidResourcePrefix)\*\*.xml" />
```

Comment:
Maybe it's too late for this change, but why support `$(MonoAndroidResourcePrefix)` at all anymore?  Consider `@(TransformFile)`,  `@(AndroidLibrary)`, `@(AndroidJavaSource)`, etc., below: *none* of those use `$(MonoAndroidResourcePrefix)`, and thus aren't susceptible to pulling in `C:\*\*`.

*Alternatively*, perhaps we could *require*/force `$(MonoAndroidResourcePrefix)` to *end with* "directory-separator-char", which would allow us to change these includes to e.g.:

```xml
<AndroidResource Include="$(MonoAndroidResourcePrefix)*\*.xml" />
```

which would *also* prevent us from trying to traverse `C:\*\*.xml`.

Reply:
MAUI uses `$(MonoAndroidResourcePrefix)` so they can redirect it to `Platforms/Android/Resources`:

https://github.com/dotnet/maui/blob/965698b1dd996ed1b062cda3b360cdaa61039ddc/src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.SingleProject.targets#L41-L42

But considering they use `EnsureTrailingSlash`:

https://github.com/dotnet/maui/blob/965698b1dd996ed1b062cda3b360cdaa61039ddc/src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.SingleProject.targets#L11-L12

So, then the only concern is a classic Xamarin.Android app has this in the project file:
```xml
<MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
<MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
```

But they should just delete these lines, and maybe it's fine to require a trailing `/`? It would make the code cleaner.